### ISSUE TITLE
Response size

### ISSUE THREAD

**Issue by dovier84** [abconlinecourses/chatgpt-xblock#5]
Is there any intentional limitation regarding the component's response?

If I use the same context & the same question directly in the openai interface, this answer is much longer than the answer given by xblocks (for the same context and the same question).

**Comment by isankadn** [comment]
Yes, we use 'max_tokens=150' to limit the fees. If it’s advisable, we can change that hardcoded value to a variable.

**Comment by dovier84** [comment]
It would be great if that value could be changed from the UI, along the others parameters.

**Comment by isankadn** [comment]
@dovier84 please check the latest, if you are still interested.
